### PR TITLE
🐛 Fixing rounding of numeric values for async destinations

### PR DESCRIPTION
**Pull request by cynthiaxyin**
For #28940:
1. Platform change (airbytehq/airbyte-platform-internal#9158)
2. Async destinations change (this PR)
3. Non-async destinations change (will be done separately)

The order (1) and (2) are merged doesn't matter. But both of them need to be merged before the problem is fixed (for async destinations).

### Problem
Previously, numeric values with many digits after the decimal point (ex: `1884518522.1684267241`) would get truncated in the `_airbyte_data` column in the raw table (ex: `1.884518522168427e+09`). We should preserve full values in the raw table (after which numeric-typed columns may have precision restrictions per destination, ex: [Snowflake has ~15 digits of precision](https://docs.snowflake.com/en/sql-reference/data-types-numeric#data-types-for-floating-point-numbers) so `1884518522.16843`).

### Solution
Deserialize the Airbyte record message string with `DeserializationFeature.USE_BIG_DECIMAL_FOR_FLOATS` to preserve all the decimals.

### Testing
Tested with source Postgres to destination Snowflake. See [old raw table](https://app.snowflake.com/xdaabwo/gz45853/#/data/databases/INTEGRATION_TEST_DESTINATION/schemas/airbyte_internal/table/cynthia_rounding_raw__stream_cynthia_rounding_rounding/data-preview), [new raw table](https://app.snowflake.com/xdaabwo/gz45853/#/data/databases/INTEGRATION_TEST_DESTINATION/schemas/airbyte_internal/table/cynthia_rounding5_raw__stream_cynthia_rounding5_rounding/data-preview), and [final table](https://app.snowflake.com/xdaabwo/gz45853/#/data/databases/INTEGRATION_TEST_DESTINATION/schemas/CYNTHIA_ROUNDING5/table/CYNTHIA_ROUNDING5_ROUNDING/data-preview).

### Reading
The main change is in `AsyncStreamConsumer.java` and `Jsons.java`, with an added unit test in `AsyncStreamConsumerTest.java`.

### CODE REVIEW

File: airbyte-integrations/connectors/destination-bigquery/build.gradle
Comment (cynthiaxyin) [inline airbyte-integrations/connectors/destination-bigquery/build.gradle:9]: Please CMIIW, my understanding of the steps are:

1. Wait for PR approval
2. Slash publish the CDK
3. Toggle this line (and the same for the other 2 dests) back to false
4. Merge the PR like usual

**Comment by airbyte-oss-build-runner** [comment]
##  destination-bigquery test report (commit [`89475c9c65`](https://github.com/airbytehq/airbyte/pull/31083/commits/89475c9c65858cd0f4786ea92c9c8953b6bae35b)) - ❌

⏲️  Total pipeline duration: 09mn15s 

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Build connector tar                                                  | ✅        |
| Build destination-bigquery docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                            | ❌        |
| Java Connector Integration Tests                                     | ✅        |
| Validate metadata for destination-bigquery                           | ✅        |
| Connector version semver check                                       | ✅        |
| Connector version increment check                                    | ✅        |
| QA checks                                                            | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/cynthia_numeric_rounding/1696894645/89475c9c65858cd0f4786ea92c9c8953b6bae35b/destination-bigquery/2.0.26/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:89475c9c65858cd0f4786ea92c9c8953b6bae35b)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-redshift test report (commit [`3c7bc10bed`](https://github.com/airbytehq/airbyte/pull/31083/commits/3c7bc10bed8cee9d285ec717d328bf6c4ce460aa)) - ✅

⏲️  Total pipeline duration: 22mn43s 

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Build connector tar                                                  | ✅        |
| Build destination-redshift docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                            | ✅        |
| Build airbyte/normalization-redshift:dev                             | ✅        |
| Java Connector Integration Tests                                     | ✅        |
| Validate metadata for destination-redshift                           | ✅        |
| Connector version semver check                                       | ✅        |
| QA checks                                                            | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/cynthia_numeric_rounding/1696895744/3c7bc10bed8cee9d285ec717d328bf6c4ce460aa/destination-redshift/0.6.8/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:3c7bc10bed8cee9d285ec717d328bf6c4ce460aa)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-redshift test
```

File: airbyte-integrations/connectors/destination-bigquery/build.gradle
Comment (edgao) [reply airbyte-integrations/connectors/destination-bigquery/build.gradle:9]: looks right to me!

File: airbyte-integrations/connectors/destination-bigquery/src/test/java/io/airbyte/integrations/destination/bigquery/CdkImportTest.java
Comment (edgao) [inline airbyte-integrations/connectors/destination-bigquery/src/test/java/io/airbyte/integrations/destination/bigquery/CdkImportTest.java:20]: @aaronsteers can we delete this test class? looks like it was something for cdk dev work

**Review by edgao — APPROVED** [review]
master cdk is already on 0.1.5 so you'll need to fix that conflict :(

we should keep an eye on [this dashboard](https://airbyte.metabaseapp.com/dashboard/234-cloud-speed-dashboard) in case this causes a noticeable drop? but I'm not expecting anything egregious - :shipit:

**Comment by airbyte-oss-build-runner** [comment]
##  destination-redshift test report (commit [`645070c69d`](https://github.com/airbytehq/airbyte/pull/31083/commits/645070c69de903b6528eff21d36e944825f18865)) - ❌

⏲️  Total pipeline duration: 21mn04s 

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Build connector tar                                                  | ✅        |
| Build destination-redshift docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                            | ✅        |
| Build airbyte/normalization-redshift:dev                             | ✅        |
| Java Connector Integration Tests                                     | ❌        |
| Validate metadata for destination-redshift                           | ✅        |
| Connector version semver check                                       | ✅        |
| QA checks                                                            | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/cynthia_numeric_rounding/1696972926/645070c69de903b6528eff21d36e944825f18865/destination-redshift/0.6.9/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:645070c69de903b6528eff21d36e944825f18865)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-redshift test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-snowflake test report (commit [`645070c69d`](https://github.com/airbytehq/airbyte/pull/31083/commits/645070c69de903b6528eff21d36e944825f18865)) - ❌

⏲️  Total pipeline duration: 12mn43s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Build connector tar                                                   | ✅        |
| Build destination-snowflake docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                             | ✅        |
| Java Connector Integration Tests                                      | ❌        |
| Validate metadata for destination-snowflake                           | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ✅        |
| QA checks                                                             | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/cynthia_numeric_rounding/1696972926/645070c69de903b6528eff21d36e944825f18865/destination-snowflake/3.2.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:645070c69de903b6528eff21d36e944825f18865)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-snowflake test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-bigquery test report (commit [`645070c69d`](https://github.com/airbytehq/airbyte/pull/31083/commits/645070c69de903b6528eff21d36e944825f18865)) - ✅

⏲️  Total pipeline duration: 07mn16s 

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Build connector tar                                                  | ✅        |
| Build destination-bigquery docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                            | ✅        |
| Java Connector Integration Tests                                     | ✅        |
| Validate metadata for destination-bigquery                           | ✅        |
| Connector version semver check                                       | ✅        |
| Connector version increment check                                    | ✅        |
| QA checks                                                            | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/cynthia_numeric_rounding/1696972926/645070c69de903b6528eff21d36e944825f18865/destination-bigquery/2.1.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:645070c69de903b6528eff21d36e944825f18865)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-bigquery test report (commit [`645070c69d`](https://github.com/airbytehq/airbyte/pull/31083/commits/645070c69de903b6528eff21d36e944825f18865)) - ✅

⏲️  Total pipeline duration: 03mn23s 

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Build connector tar                                                  | ✅        |
| Build destination-bigquery docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                            | ✅        |
| Java Connector Integration Tests                                     | ✅        |
| Validate metadata for destination-bigquery                           | ✅        |
| Connector version semver check                                       | ✅        |
| Connector version increment check                                    | ✅        |
| QA checks                                                            | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/cynthia_numeric_rounding/1696976020/645070c69de903b6528eff21d36e944825f18865/destination-bigquery/2.1.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:645070c69de903b6528eff21d36e944825f18865)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-snowflake test report (commit [`645070c69d`](https://github.com/airbytehq/airbyte/pull/31083/commits/645070c69de903b6528eff21d36e944825f18865)) - ✅

⏲️  Total pipeline duration: 11mn18s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Build connector tar                                                   | ✅        |
| Build destination-snowflake docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                             | ✅        |
| Java Connector Integration Tests                                      | ✅        |
| Validate metadata for destination-snowflake                           | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ✅        |
| QA checks                                                             | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/cynthia_numeric_rounding/1696976020/645070c69de903b6528eff21d36e944825f18865/destination-snowflake/3.2.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:645070c69de903b6528eff21d36e944825f18865)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-snowflake test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-redshift test report (commit [`645070c69d`](https://github.com/airbytehq/airbyte/pull/31083/commits/645070c69de903b6528eff21d36e944825f18865)) - ✅

⏲️  Total pipeline duration: 20mn37s 

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Build connector tar                                                  | ✅        |
| Build destination-redshift docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                            | ✅        |
| Build airbyte/normalization-redshift:dev                             | ✅        |
| Java Connector Integration Tests                                     | ✅        |
| Validate metadata for destination-redshift                           | ✅        |
| Connector version semver check                                       | ✅        |
| QA checks                                                            | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/cynthia_numeric_rounding/1696978409/645070c69de903b6528eff21d36e944825f18865/destination-redshift/0.6.9/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:645070c69de903b6528eff21d36e944825f18865)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-redshift test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-snowflake test report (commit [`645070c69d`](https://github.com/airbytehq/airbyte/pull/31083/commits/645070c69de903b6528eff21d36e944825f18865)) - ✅

⏲️  Total pipeline duration: 03mn22s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Build connector tar                                                   | ✅        |
| Build destination-snowflake docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                             | ✅        |
| Java Connector Integration Tests                                      | ✅        |
| Validate metadata for destination-snowflake                           | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ✅        |
| QA checks                                                             | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/cynthia_numeric_rounding/1696978409/645070c69de903b6528eff21d36e944825f18865/destination-snowflake/3.2.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:645070c69de903b6528eff21d36e944825f18865)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-snowflake test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-bigquery test report (commit [`645070c69d`](https://github.com/airbytehq/airbyte/pull/31083/commits/645070c69de903b6528eff21d36e944825f18865)) - ✅

⏲️  Total pipeline duration: 03mn20s 

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Build connector tar                                                  | ✅        |
| Build destination-bigquery docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                            | ✅        |
| Java Connector Integration Tests                                     | ✅        |
| Validate metadata for destination-bigquery                           | ✅        |
| Connector version semver check                                       | ✅        |
| Connector version increment check                                    | ✅        |
| QA checks                                                            | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/cynthia_numeric_rounding/1696978409/645070c69de903b6528eff21d36e944825f18865/destination-bigquery/2.1.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:645070c69de903b6528eff21d36e944825f18865)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-redshift test report (commit [`1059004942`](https://github.com/airbytehq/airbyte/pull/31083/commits/10590049427ef8ac0cf95e760f8cb264ffdbec4b)) - ✅

⏲️  Total pipeline duration: 21mn11s 

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Build connector tar                                                  | ✅        |
| Build destination-redshift docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                            | ✅        |
| Build airbyte/normalization-redshift:dev                             | ✅        |
| Java Connector Integration Tests                                     | ✅        |
| Validate metadata for destination-redshift                           | ✅        |
| Connector version semver check                                       | ✅        |
| QA checks                                                            | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/cynthia_numeric_rounding/1696981402/10590049427ef8ac0cf95e760f8cb264ffdbec4b/destination-redshift/0.6.9/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:10590049427ef8ac0cf95e760f8cb264ffdbec4b)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-redshift test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-bigquery test report (commit [`1059004942`](https://github.com/airbytehq/airbyte/pull/31083/commits/10590049427ef8ac0cf95e760f8cb264ffdbec4b)) - ✅

⏲️  Total pipeline duration: 07mn45s 

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Build connector tar                                                  | ✅        |
| Build destination-bigquery docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                            | ✅        |
| Java Connector Integration Tests                                     | ✅        |
| Validate metadata for destination-bigquery                           | ✅        |
| Connector version semver check                                       | ✅        |
| Connector version increment check                                    | ✅        |
| QA checks                                                            | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/cynthia_numeric_rounding/1696981402/10590049427ef8ac0cf95e760f8cb264ffdbec4b/destination-bigquery/2.1.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:10590049427ef8ac0cf95e760f8cb264ffdbec4b)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-bigquery test report (commit [`1059004942`](https://github.com/airbytehq/airbyte/pull/31083/commits/10590049427ef8ac0cf95e760f8cb264ffdbec4b)) - ✅

⏲️  Total pipeline duration: 03mn43s 

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Build connector tar                                                  | ✅        |
| Build destination-bigquery docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                            | ✅        |
| Java Connector Integration Tests                                     | ✅        |
| Validate metadata for destination-bigquery                           | ✅        |
| Connector version semver check                                       | ✅        |
| Connector version increment check                                    | ✅        |
| QA checks                                                            | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/cynthia_numeric_rounding/1696987447/10590049427ef8ac0cf95e760f8cb264ffdbec4b/destination-bigquery/2.1.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:10590049427ef8ac0cf95e760f8cb264ffdbec4b)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-snowflake test report (commit [`1059004942`](https://github.com/airbytehq/airbyte/pull/31083/commits/10590049427ef8ac0cf95e760f8cb264ffdbec4b)) - ❌

⏲️  Total pipeline duration: 21mn13s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Build connector tar                                                   | ✅        |
| Build destination-snowflake docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                             | ✅        |
| Java Connector Integration Tests                                      | ❌        |
| Validate metadata for destination-snowflake                           | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ✅        |
| QA checks                                                             | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/cynthia_numeric_rounding/1696987447/10590049427ef8ac0cf95e760f8cb264ffdbec4b/destination-snowflake/3.2.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:10590049427ef8ac0cf95e760f8cb264ffdbec4b)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-snowflake test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-redshift test report (commit [`1059004942`](https://github.com/airbytehq/airbyte/pull/31083/commits/10590049427ef8ac0cf95e760f8cb264ffdbec4b)) - ✅

⏲️  Total pipeline duration: 03mn12s 

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Build connector tar                                                  | ✅        |
| Build destination-redshift docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                            | ✅        |
| Build airbyte/normalization-redshift:dev                             | ✅        |
| Java Connector Integration Tests                                     | ✅        |
| Validate metadata for destination-redshift                           | ✅        |
| Connector version semver check                                       | ✅        |
| QA checks                                                            | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/cynthia_numeric_rounding/1696987447/10590049427ef8ac0cf95e760f8cb264ffdbec4b/destination-redshift/0.6.9/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:10590049427ef8ac0cf95e760f8cb264ffdbec4b)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-redshift test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-redshift test report (commit [`1059004942`](https://github.com/airbytehq/airbyte/pull/31083/commits/10590049427ef8ac0cf95e760f8cb264ffdbec4b)) - ✅

⏲️  Total pipeline duration: 03mn32s 

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Build connector tar                                                  | ✅        |
| Build destination-redshift docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                            | ✅        |
| Build airbyte/normalization-redshift:dev                             | ✅        |
| Java Connector Integration Tests                                     | ✅        |
| Validate metadata for destination-redshift                           | ✅        |
| Connector version semver check                                       | ✅        |
| QA checks                                                            | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/cynthia_numeric_rounding/1696989283/10590049427ef8ac0cf95e760f8cb264ffdbec4b/destination-redshift/0.6.9/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:10590049427ef8ac0cf95e760f8cb264ffdbec4b)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-redshift test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-bigquery test report (commit [`1059004942`](https://github.com/airbytehq/airbyte/pull/31083/commits/10590049427ef8ac0cf95e760f8cb264ffdbec4b)) - ✅

⏲️  Total pipeline duration: 03mn10s 

| Step                                                                 | Result   |
|:---------------------------------------------------------------------|:---------|
| Build connector tar                                                  | ✅        |
| Build destination-bigquery docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                            | ✅        |
| Java Connector Integration Tests                                     | ✅        |
| Validate metadata for destination-bigquery                           | ✅        |
| Connector version semver check                                       | ✅        |
| Connector version increment check                                    | ✅        |
| QA checks                                                            | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/cynthia_numeric_rounding/1696989283/10590049427ef8ac0cf95e760f8cb264ffdbec4b/destination-bigquery/2.1.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:10590049427ef8ac0cf95e760f8cb264ffdbec4b)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-bigquery test
```

**Comment by airbyte-oss-build-runner** [comment]
##  destination-snowflake test report (commit [`1059004942`](https://github.com/airbytehq/airbyte/pull/31083/commits/10590049427ef8ac0cf95e760f8cb264ffdbec4b)) - ✅

⏲️  Total pipeline duration: 11mn06s 

| Step                                                                  | Result   |
|:----------------------------------------------------------------------|:---------|
| Build connector tar                                                   | ✅        |
| Build destination-snowflake docker image for platform(s) linux/x86_64 | ✅        |
| Java Connector Unit Tests                                             | ✅        |
| Java Connector Integration Tests                                      | ✅        |
| Validate metadata for destination-snowflake                           | ✅        |
| Connector version semver check                                        | ✅        |
| Connector version increment check                                     | ✅        |
| QA checks                                                             | ✅        |

🔗 [View the logs here](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/cynthia_numeric_rounding/1696989283/10590049427ef8ac0cf95e760f8cb264ffdbec4b/destination-snowflake/3.2.1/output.html)

☁️ [View runs for commit in Dagger Cloud](https://alpha.dagger.cloud/changeByPipelines?filter=dagger.io/git.ref:10590049427ef8ac0cf95e760f8cb264ffdbec4b)

*Please note that tests are only run on PR ready for review. Please set your PR to draft mode to not flood the CI engine and upstream service on following commits.*
**You can run the same pipeline locally on this branch with the [airbyte-ci](https://github.com/airbytehq/airbyte/blob/master/airbyte-ci/connectors/pipelines/README.md) tool with the following command**
```bash
airbyte-ci connectors --name=destination-snowflake test
```

**Pull request merged**
